package.json

```diff
@@ -37,6 +37,7 @@
     "@types/jest": "^24.0.15",
     "@types/lolex": "^3.1.1",
     "@types/nodemailer": "^4.6.8",
+    "@types/redis": "^2.8.28",
```

Comment:
I think @types/redis can be removed with the new library version.

Reply:
[2938b47](https://github.com/pagopa/io-functions-service-messages/pull/57/commits/2938b4711b9f2d94dc202af1441bf1e910377dd1)